contracts/lib/SafeMath.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.4.15;
```

Comment:
Shouldn't we use the version controlled version of this using npm?

Reply:
good practice to compile everything with the same compiler